utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
On Tue, May 17, 2016 at 01:05:45PM -0700, Michael Crosby wrote:
> Are you asking a question or what are you trying to get at? 

I'm trying to decide how you figure out the exit code for the
container process.  It sounds like you're saying:

  “The runtime will not track or provide access to this information.
  The process that called ‘create’ should setup its own wait to
  collect that information and call ‘delete’.”

wait(1p) is a fairly convenient way to do that from the shell, so I'm
fine with this approach, but it is shifting that reap/cleanup
functionality from the runtime (which used to handle it) to the
caller, so we probably want clear docs to this effect.